bindata/manifests/daemon/daemonset.yaml

```diff
@@ -11,6 +11,8 @@ spec:
     matchLabels:
       app: sriov-network-config-daemon
   updateStrategy:
+    rollingUpdate:
+      maxUnavailable: 25%
```

Comment:
Thanks for the comments!
Updated.

Reply:
Hmm, I tried to verify this PR by running the tests. It will fail the test unless this value is set to either 10% or 33%. So neither 25% nor 1 would work.

Reply:
Should we set it to 33% instead?

Reply:
Updated to 33%.